library/HTMLPurifier/HTML5Config.php

```diff
@@ -60,9 +60,9 @@ public static function inherit(HTMLPurifier_Config $config)
 
     /**
      * @param HTMLPurifier_ConfigSchema $schema
-     * @param HTMLPurifier_PropertyList $parent OPTIONAL
+     * @param HTMLPurifier_PropertyList|null $parent OPTIONAL
      */
-    public function __construct(HTMLPurifier_ConfigSchema $schema, HTMLPurifier_PropertyList $parent = null)
+    public function __construct(HTMLPurifier_ConfigSchema $schema, $parent = null)
```

Comment:
An additional check for proper `$parent` type would be nice, something like:

```php
if ($parent && !$parent instanceof HTMLPurifier_PropertyList) {
    throw new InvalidArgumentException(
        'Argument #2 ($parent) passed to ' . __METHOD__ . '() must be an instance of HTMLPurifier_PropertyList or null'
    );
}
```

Reply:
Done :)